app/src/main/java/com/example/pocketcloset/Fragments/OutfitsFragment.java

```diff
@@ -51,15 +55,94 @@ public static OutfitsFragment newInstance(String param1, String param2) {
+        user = getArguments().getParcelable("userToFilterBy");
+        return inflater.inflate(R.layout.fragment_outfits, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        allOutfits = new ArrayList<>();
+        adapter = new OutfitsAdapter(getContext(), allOutfits);
+        rvOutfits = view.findViewById(R.id.rvOutfits);
+
+        rvOutfits.setAdapter(adapter);
+        // set the layout manager on the recycler view
+        rvOutfits.setLayoutManager(new LinearLayoutManager(getContext()));
+
+
+        GridLayoutManager gridLayoutManager = new GridLayoutManager(
+                getContext(),
+                2 /* numberOfColumns */);
+
+        rvOutfits.setLayoutManager(gridLayoutManager);
+
+
+        swipeContainer = view.findViewById(R.id.swipeContainer);
+        //Setup refresh listener which triggers new data loading
+        swipeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                adapter.clear();
+                queryOutfits();
+                adapter.notifyDataSetChanged();
+                swipeContainer.setRefreshing(false);
+            }
+        });
+
+        swipeContainer.setColorSchemeResources(android.R.color.holo_blue_bright,
+                android.R.color.holo_green_light,
+                android.R.color.holo_orange_light,
+                android.R.color.holo_red_light);
+
+
+        queryOutfits();
+    }
+
+    protected void queryOutfits() {
+        // specify what type of data we want to query - Post.class
+        ParseQuery<Outfit> query = ParseQuery.getQuery(Outfit.class);
+        // include data referred by user key
+        Log.i("WardrobeFragment", "queryClothes(with params) ran");
+        query.include(Outfit.KEY_OUTFIT_NAME);
+        query.include(Outfit.KEY_BOTTOMS);
+        query.include(Outfit.KEY_TOP);
+        query.include(Outfit.KEY_BRACELET);
+        query.include(Outfit.KEY_NECKWEAR);
+        query.include(Outfit.KEY_SHOES);
+        query.include(Outfit.KEY_OVERWEAR);
+        query.include(Outfit.KEY_HANDHELD);
+        query.include(Outfit.KEY_EARRINGS);
+        query.include(Outfit.KEY_HEADWEAR);
+        query.whereEqualTo(Outfit.KEY_USER, user);
+
+        // limit query to latest 20 items
+        query.setLimit(20);
+        // order posts by creation date (newest first)
+        query.addDescendingOrder("createdAt");
+        // start an asynchronous call for posts
+        query.findInBackground(new FindCallback<Outfit>() {
+            @Override
+            public void done(List<Outfit> outfits, ParseException e) {
+                // check for errors
+                if (e != null) {
+                    Log.e("WardrobeFragment", "Issue with getting clothing", e);
```

Comment:
you will eventually want to show an error message in these cases. I would recommend putting in a TODO with an easily-grepable string e.x. TODO add an error message

This way you can easily grep your code for the TODOs when you get around to adding the error messages, with a much lower chance of missing a case.